### DISCUSSION TITLE
Plex enhanced app recent on deck does not change values

### DISCUSSION THREAD

**Discussion by goku-son** [linuxserver/Heimdall#743] [App issues]
The plex enhanced app shows 50 for recent and 0 from on deck both of which are inaccurate and never change. I tried deleting and setting up the app again, but it just brings back the same values. 

Any ideas as to what may be the issue?

Thanks in advance.

**Comment by mbedez** [comment]
Same problem here.

**Comment by ZoltrixGFC** [comment]
Yep same here as well.
